src/plugins/twitter/twitter-fr.hbs

```diff
@@ -17,21 +17,39 @@
 	<p>Ce plugiciel aide à afficher des chronologies intégrées Twitter.</p>
 </section>
 
-<div lang="en">
-<p><strong>Needs translation</strong></p>
 <section>
-	<h2>Example: Embedded User Timeline</h2>
-	<div class="wb-twitter">
-		<a class="twitter-timeline" href="https://twitter.com/WebExpToolkit" data-tweet-limit="3">Tweets by @WebExpToolkit</a>
-	</div>
+	<h2>Exemples&nbsp;: Chronologie de profil</h2>
+
+	<div class="row">
+		<section class="col-md-6">
+			<h3>Compte standard</h3>
+			<div class="wb-twitter">
+				<a class="twitter-timeline" href="https://twitter.com/WebExpToolkit" data-tweet-limit="3">Tweets de @WebExpToolkit</a>
+			</div>
+			<section>
+				<h4>Code</h4>
+				<pre><code>&lt;div class=&quot;wb-twitter&quot;&gt;
+	&lt;a class=&quot;twitter-timeline&quot; href=&quot;https://twitter.com/WebExpToolkit&quot; data-tweet-limit=&quot;3&quot; &gt;Tweets de @WebExpToolkit&lt;/a&gt;
+&lt;/div&gt;</code></pre>
+			</section>
+		</section>
 
-	<section>
-		<h3>Code</h3>
-		<pre><code>&lt;div class=&quot;wb-twitter&quot;&gt;
-&lt;a class=&quot;twitter-timeline&quot; href=&quot;https://twitter.com/WebExpToolkit&quot; data-tweet-limit=&quot;3&quot; &gt;Tweets by @WebExpToolkit&lt;/a&gt;
+		<section class="col-md-6">
+			<h3>Compte «&nbsp;certifié&nbsp;»</h3>
+			<div class="wb-twitter">
+				<a class="twitter-timeline" href="https://twitter.com/Canada" data-tweet-limit="2">Tweets de @Canada</a>
```

Comment:
I would've preferred using [@AuCanada](https://twitter.com/AuCanada) (i.e. official French version of [@Canada](https://twitter.com/Canada)) for this example, but... Twitter doesn't currently classify it as a "verified" government account.

[@AuCanada](https://twitter.com/AuCanada)'s widget works the same way as a standard account and trying to view the account's profile shows a login wall. So it can't be used as a "verified" example at this time :(.